bindings/go/credentials/graph_test.go

```diff
@@ -187,11 +187,24 @@ func GetGraph(t testing.TB, yaml string) (credentials.GraphResolver, error) {
 							"username": "test1",
 							"password": "bar",
 						}, nil
+					case "notfound.io":
+						return nil, nil
 					default:
 						return nil, fmt.Errorf("failed access")
 					}
 				},
 			}, nil
+		case "ErrorPluginResolve":
```

Comment:
I don't think that's a type... right? 🤔 Like is this a valid value for `repoType.GetType().String() {`?